src/main/java/org/cloudfoundry/promregator/cfaccessor/CFAccessorCacheClassic.java

```diff
@@ -1,396 +1,392 @@
-package org.cloudfoundry.promregator.cfaccessor;
```

Comment:
You converted the entire file from newline format `lf` (which is the default for Promregator) to `cflf`. This isn't good 😬

Reply:
Yes, have been having some wierd issues. Should be fixed now